File: wrappers/java/kendo-taglib/src/main/java/com/kendoui/taglib/json/Template.java
package com.kendoui.taglib.json;

import java.io.IOException;
import java.io.Writer;

public class Template extends Function {
    
    public Template(String body) {
        super(body);
    }

    public void write(Writer out) throws IOException {
        out.append("kendo.template($('#" + getBody()  + "').html())");
    }
}

File: wrappers/php/lib/Kendo/Template.php
<?php

namespace Kendo;

class Template extends JavaScriptFunction {
    private $value;

    function __construct($value) {
        $this->value = $value;
    }

    public function value() {
        return "kendo.template($('\\#".$this->value."').html())";
    }
}
?>
